tools/i18nExtractKeys.vite.js

```diff
@@ -0,0 +1,38 @@
+import path from 'path';
+
+const uniqueKeys = new Set();
+
+function extractRegexPlugin({regex, extraKeys, fileOutput}) {
+	fileOutput =
+		fileOutput || path.join('locale', 'uiTranslationKeysBackend.json');
```

Comment:
I'm not sure it's good to touch the `locale` folder, so I'd rather leave this file somewhere else.

I saw that you've probably planning to run this extractor within plugins, but I guess it's not required for a plugin to have a `locale` folder.

Reply:
Yeah, I am not fan of that neither. I went easy route as the registered paths in Locale.php are pointing to the locale folder. @asmecher mentioned registry folder might be good fit, so I will adjust it to that until someone comes with some other/better suggestion :-).